### ISSUE TITLE
Question on group options. Should not group rather be an Object than String?

### ISSUE THREAD

**Issue by polinom** [SortableJS/polymer-sortablejs#9]
Should not `group` property be a type of 'Object' instead of 'String'?

https://github.com/SortableJS/polymer-sortablejs/blob/master/polymer-sortablejs.html#L15

Just in case if user want's provide object as stated in documentation:

```javascript
{ group: "name",  // or { name: "...", pull: [true, false, clone], put: [true, false, array] }}
```

**Comment by christophe-g** [comment]
#14

**Comment by gutenye** [comment]
fixed by #14
